blog/_posts/2017-12-11-Thanks-for-diversifying-tech-with-us.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Thanks for diversifying tech with us!
+layout: post
+created_at: Mon December 11 2017
+permalink: blog/2017-12-11-Thanks-for-diversifying-tech-with-us
+current: blog
+author: Inês Coelho
+twitter: ines_opcoelho
+categories:
+- news
+---
+
+For the past five years, every (north hemisphere) summer, RGSoC has been offering the opportunity for women (transgender and cisgender) and non-binary people from all over the world to work on an Open Source project. This wouldn't be possible without the help and love of our wonderful community. 
+
+![Together we are changing the world!](/img/blog/2017/2017-12-12-change-the-world.jpg)
+<div class="image-credits">Our 20 teams - the lives we touched on the 2017 edition of RGSoC (Image: Inês Coelho)</div>
+
+### Our sponsors and donors
+
+Our program is based on a crowdfunding and sponsorship campaign that runs all year round, whose money is used to finance our teams. This year we were able to fund 16 out of our 20 participating teams of two! 
+
+We can't thank enough all the companies that [sponsored](https://railsgirlssummerofcode.org/sponsors/) RGSoC: we wouldn't be here without you! A special thank you goes to our amazing partners, [Travis CI](https://travis-ci.com/) and [GitHub](https://github.com/), our platinum sponsor [Nokia](https://www.nokia.com) and our gold sponsors, [Google Open Source](https://opensource.google.com/) and [Malwarebytes](https://www.malwarebytes.com/). 
+
+We are also extremely grateful to the 161 individuals who donated to our campaign. You are the best! 
+
+And, of course, a huge thank you to all the [conferences](https://railsgirlssummerofcode.org/students/conferences/#conferences) that welcomed our participants and donated tickets to our teams! You are awesome!
```

Comment:
I'd try to tone down a bit on the exclamation marks, so maybe change to “[...] welcomed our participants and donated tickets to our teams — you are awesome!”